paasta_tools/utils.py

```diff
@@ -246,7 +245,7 @@ class InvalidInstanceConfig(Exception):
 
 Constraint = Sequence[str]
 
-# e.g. ['GROUP_BY', 'habitat', 2]. Marathon doesn't like that so we'll convert to Constraint later.
+# e.g. ['GROUP_BY', 'habitat', 2]. Tron doesn't like that so we'll convert to Constraint later.
 UnstringifiedConstraint = Sequence[Union[str, int, float]]
 
 SecurityConfigDict = Dict  # Todo: define me.
```

Comment:
i bet this is only used for the firewall code, so if we delete that this can probably go too